include/SugarFolders/SugarFolders.php

```diff
@@ -42,21 +42,26 @@
     die('Not A Valid Entry Point');
 }
 
-require_once('include/ytree/Tree.php');
-require_once('include/ytree/ExtNode.php');
-
-
-class SugarFolderEmptyException extends Exception
-{
-}
+require_once(__DIR__ . '/../ytree/Tree.php');
+require_once(__DIR__ . '/../ytree/ExtNode.php');
+require_once(__DIR__ . '/SugarFolderEmptyException.php');
+require_once(__DIR__ . '/../TimeDate.php');
 
 /**
  * Polymorphic buckets - place any item in a folder
  */
 class SugarFolder
 {
-
-    // public attributes
+    // Protected globals
+    protected $currentUser     = null;
```

Comment:
please use @var [type] in phpdoc, above the variable.

Reply:
Fixed